plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/_core.py

```diff
@@ -375,3 +281,49 @@ def normalize_key(key: str, str_to_replace: str, str_to_replace_with: str) -> st
     ]
     new_key = str_to_escape.join(updated_splits).replace("\\", "")
     return new_key
+
+
+def create_range_param_using_interval_override(override):
+    key = override.get_key_element()
+    value = override.value()
+    param = {
+        "name": key,
+        "type": "range",
+        "bounds": [value.start, value.end],
+    }
+    return param
+
+
+def create_choice_param_from_choice_override(override):
+    key = override.get_key_element()
+    value = override.value()
+    param = {
+        "name": key,
+        "type": "choice",
+        "values": value.list,
```

Comment:
not good. look at the example plugin. use override.sweep_string_iterator(). This will respect shuffling and glob and in general is the right way to access fields in all in choice() and range().

Reply:
`override.sweep_string_iterator()` returns a list of strings (does not respect the datatype)